### PR TITLE
Proxyless prompt building

### PR DESCRIPTION
**Pull request by capaj**

### CODE REVIEW

File: dist/LangtailNode.d.mts [hunk 1]
```diff
@@ -0,0 +1,82 @@
+import * as Core from 'openai/core';
```
Comment (rychlis) [inline dist/LangtailNode.d.mts:1]: are the dist files commited to git on purpose?

File: src/LangtailPrompts.ts [hunk 2]
```diff
@@ -143,4 +151,36 @@ export class LangtailPrompts {
...
     return result
   }
+
+  async get({
+    prompt,
+    environment,
+    version,
+  }: {
+    prompt: string
+    environment: Environment
```
Comment (rychlis) [inline src/LangtailPrompts.ts:161]: environment should be optional and default to production like invoke

Comment (capaj) [reply src/LangtailPrompts.ts]: fixed

File: dist/LangtailNode.d.mts (hunk 1 above)
Comment (capaj) [reply dist/LangtailNode.d.mts:1]: yes they are.
Will remove them-we don't need  them anymore in git. Will use beta release of SDK instead

Comment (capaj) [reply dist/LangtailNode.d.mts]: removed now

**Pull request merged**